Первый ПР к консольному приложению

Mereinalamone opened the pull request:
Первый ПР к консольному приложению калькулятор счёта

Mereinalamone committed:
Первый комит к калькулятору счёта

gusar-off commented on src/main/java/Main.java:
```diff
@@ -1,6 +1,28 @@
+        while (true) {
+            try {
+                Scanner scanner = new Scanner(System.in);
+                numberPeople = scanner.nextInt();
```
⚠️ Если передать в этот момент произвольную строку программа завершит работу с исключением. Перед тем, как прочитать инт из сканнера, нужно сначала проверить возможно ли это при помощи соответствующего метода `hasNextInt`

gusar-off commented on src/main/java/DivisionCheck.java:
```diff
@@ -0,0 +1,77 @@
+            while(true) {
+                try{
+                    System.out.printf("Введите название товара №%d: ", i);
+                    Scanner scanner = new Scanner(System.in);
```
Лучшее переиспользовать инстанс `Scanner`, а не создавать каждый раз новый.

gusar-off commented on src/main/java/DivisionCheck.java:
```diff
@@ -0,0 +1,77 @@
+                try{
+                    System.out.printf("Введите цену товара №%d: ", i);
+                    Scanner scanner = new Scanner(System.in);
+                    price = scanner.nextFloat();
```
Тоже самое, что и с hasNextInt только тут надо hasNextFloat

gusar-off commented on src/main/java/DivisionCheck.java:
```diff
@@ -0,0 +1,77 @@
+        }
+        System.out.println("Список добавленных товаров");
+        System.out.println("|Наименование товара |Стоимость |");
+        for (i = 0; i < productList.size(); i = i + 1) {
```
Можно воспользоваться циклом foreach, было бы удобнее, подробнее можно почитать тут
https://javarush.com/groups/posts/for-each-java

Mereinalamone committed:
Исправил замечания:
1) использовал методы hasNext() для обработки введённой пользователем инфармации
2) теперь переиспользуется инстанс Scanner, без пересоздания в каждом цикле как было
